js_modules/dagit/tox.ini

```diff
@@ -22,15 +22,16 @@ commands =
     yarn install
   yarn workspace @dagster-io/dagit-core generate-graphql
   yarn workspace @dagster-io/dagit-core generate-perms
-  yarn workspace @dagster-io/dagit-app lint
   yarn workspace @dagster-io/dagit-app ts
+  yarn workspace @dagster-io/dagit-app lint
   yarn workspace @dagster-io/dagit-core ts
+  yarn workspace @dagster-io/dagit-core lint
+  yarn workspace @dagster-io/ui ts
+  yarn workspace @dagster-io/ui lint
```

Comment:
added missing `lint` call for @dagster-io/core and also moved all the ts/lint checks to happen before we run any tests.